### ISSUE TITLE
Add comment block to top of TODO file?

### ISSUE THREAD

**Issue by ndushay** [richseviora/eslint-takeoff#1]
🎉  Thanks for this!

I wondered if it would be easy to add a comment block to the top of the todo file indicating it was autogenerated -- always a good warning to folks thinking of editing the file manually.

As I'm sure you know, Rubocop does:

```
# This configuration was generated by
# `rubocop --auto-gen-config`
# on 2018-09-27 15:46:41 -0700 using RuboCop version 0.59.0.
# The point is for the user to remove these configuration records
# one by one as the offenses are removed from the code base.
# Note that changes in the inspected code, or installation of new
# versions of RuboCop, may require this file to be generated again.
```

**Comment by richseviora** [comment]
So I don't know why I didn't get this notification in my mailbox but I'm seeing them now! Easy... yes! Or should be. I think I'd have to first generate the list and then prepend the comments above. I'll see if I can spend some time to do that soonish :)

**Comment by ndushay** [comment]
No rush on my account - just some feedback I wanted to provide.

**Comment by richseviora** [comment]
Implemented in version 0.0.3!
